tools/cldr-apps/src/main/webapp/js/new/cldrLocaleMap.js

```diff
@@ -0,0 +1,153 @@
+"use strict";
+
+/**
+ * @param aLocmap the map object from json
+ */
+function LocaleMap(aLocmap) {
+  this.locmap = aLocmap;
```

Comment:
Note: "it's no longer possible to reference the window object through _this_ inside a strict mode function" -- so this file needs modernizing. This file isn't used, though, if USE_DOJO is true. There's still a lot that won't work yet if USE_DOJO is false.

Reply:
It should be easy to turn LocaleMap into a class with a constructor(), and simplify the syntax, no more .prototype.